pkgs/by-name/co/codd/generated.nix

```diff
@@ -0,0 +1,113 @@
+# This was generated with cabal2nix but contains important modifications like jailbreaking haxl
+# and disabling tests selectively, among a few other minor things.
+# Don't just blindly replace it with something generated by cabal2nix.
+{
+  lib,
+  haskell,
+  mkDerivation,
+  fetchFromGitHub,
+  haskellPackages,
+}:
+
+let
+
+  # Haxl has relatively tight version requirements and is thus marked as broken.
+  haxlJailbroken = haskell.lib.markUnbroken (haskell.lib.doJailbreak haskellPackages.haxl);
+
+in
+mkDerivation rec {
+  pname = "codd";
+
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "mzabani";
+    repo = "codd";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-hFAo0DLhxULeib3OMjfTYBQIBJ5+apWlMDGsK5gUwho=";
+  };
+
+  isLibrary = false;
+
+  isExecutable = true;
+
+  libraryHaskellDepends = with haskellPackages; [
+    aeson
+    aeson-pretty
+    ansi-terminal
+    attoparsec
+    base
+    bytestring
+    clock
+    containers
+    dlist
+    filepath
+    formatting
+    hashable
+    haxlJailbroken
+    mtl
+    network-uri
+    postgresql-libpq
+    postgresql-simple
+    resourcet
+    streaming
+    text
+    time
+    transformers
+    unliftio
+    unliftio-core
+    unordered-containers
+    vector
+  ];
+
+  executableHaskellDepends = with haskellPackages; [
+    base
+    optparse-applicative
+    postgresql-simple
+    text
+    time
+  ];
+
+  testHaskellDepends = with haskellPackages; [
+    aeson
+    attoparsec
+    base
+    containers
+    hashable
+    hspec
+    hspec-core
+    mtl
+    network-uri
+    postgresql-simple
+    QuickCheck
+    random
+    resourcet
+    streaming
+    text
+    time
+    typed-process
+    unliftio
+  ];
+
```

Comment:
```suggestion
  testToolDepends = [ haskellPackages.hspec-discover ];
```

Reply:
Thanks! I'm curious what this changes, as it seemed tests would run even if I didn't declare hspec-discover as a dependency.

Reply:
It only matters if you set strictDeps = true. I'm currently trying `strictDepsByDefault = true`, which has already found a lot of issues like this.